docs/zz_generated.seed.yaml

```diff
@@ -240,6 +240,8 @@ spec:
       replicas: null
       resources: null
       tolerations: null
+  # EnableClustersMetering enables the metering on user clusters across the seed.
+  enableClustersMetering: false
```

Comment:
Do we anticipate further options coming along the way? If so, we might want to plan a structure like

```
spec:
  metering:
    enabled: true|false
```

In any case, I would not call it "ClustersMetering", but "ClusterMetering" or even "UserclusterMetering".

Reply:
Actually yes, I wanted to add this structure in the following up PR that I am working on now, which is the integration in the API, because I would have a better overview of which fields and configurations that could be added in this spec. Regarding the naming, since the metering is simply on a seed level, I guess `MeteringSpec` is convenient.